cognite/client/_api/units.py

```diff
@@ -41,7 +38,7 @@ def retrieve(self, external_id: MutableSequence[str], ignore_unknown_ids: bool =
     def retrieve(
         self, external_id: str | MutableSequence[str], ignore_unknown_ids: bool = False
     ) -> Unit | UnitList | None:
-        """`Retrieve one or more unit <https://pr-50.units-api.preview.cogniteapp.com/#tag/Units/operation/retrieve_units_by_ids_api_v1_projects__project__units_byids_post>`_
+        """`Retrieve one or more unit <https://developer.cognite.com/api#tag/Units/operation/byIdsUnits>`_
```

Comment:
None of these urls seem to work yet?

Reply:
no, but they will be there once we merge https://github.com/cognitedata/service-contracts/pull/2375

Reply:
now they work @haakonvt